include/k4arecord/record.h

```diff
@@ -202,6 +358,46 @@ K4ARECORD_EXPORT k4a_result_t k4a_record_write_capture(k4a_record_t recording_ha
  */
 K4ARECORD_EXPORT k4a_result_t k4a_record_write_imu_sample(k4a_record_t recording_handle, k4a_imu_sample_t imu_sample);
 
+/** Writes data for a custom track to file.
+ *
+ * \param recording_handle
+ * The handle of a new recording, obtained by k4a_record_create().
+ *
+ * \param track_name
+ * The name of the custom track that the data is going to be written to.
+ *
+ * \param timestamp_usec
+ * The timestamp in microseconds for the custom track data.
+ *
```

Comment:
This timestamp should be namespaced

Reply:
Please apply this throughout where possible

Reply:
The recording files only have a single timestamp, which for captures ends up recording device timestamp.
Custom track data may not even come from a device, and the timestamp meaning is just user-defined. I'm not sure it makes sense to namespace these timestamps as device/system timestamps.

Reply:
Playback API's basically mirror the SDK api's to get capture, images, and image info. The timestamp is one of them. Currently with the SDK the user can get two timestamps a device timestamp and a system timestamp. At some point you will need to add system timestamp and when that happens users will need to know the difference. So we should clarify what timestamp is being returned here and throughout. In 1.1 we deprecated k4a_image_get_timestamp_usec in favor of a new API k4a_image_get_device_timestamp_usec.